UnityProject/Assets/Scripts/UI/Objects/Shuttles/GUI_ShuttleControl.cs

```diff
@@ -302,17 +306,19 @@ private IEnumerator Refresh()
 			EntryList.RefreshTrackedPos();
 			//Logger.Log((MatrixMove.shuttleFuelSystem.FuelLevel * 100).ToString());
 			var fuelGauge = (NetUIElement<string>)this["FuelGauge"];
-			if (MatrixMove.ShuttleFuelSystem == null)
+			if (FuelSystemReference == null)
 			{
 				if (fuelGauge.Value != "100")
 				{
 					fuelGauge.SetValueServer((100).ToString());
 				}
 
 			}
-			else
+			else if(FuelSystemReference.Connector != null && FuelSystemReference.Connector.canister != null
+			&& FuelSystemReference.Connector.canister.GasContainer != null)
 			{
-				fuelGauge.SetValueServer(Math.Round((MatrixMove.ShuttleFuelSystem.FuelLevel * 100)).ToString());
+				string value = $"{(FuelSystemReference.FuelLevel / 100f * FuelSystemReference.Connector.canister.GasContainer.MaximumMoles)}";
```

Comment:
```suggestion
				string value = $"{(FuelSystemReference.FuelLevel * 100f)}";
```
`FuelLevel` is already the ratio of the amount of fuel to the maximum capacity. Just multiply by 100 and it should work, shouldn't it?

Reply:
I'll take a look at it in a few hours